app/controllers/coronavirus_form/check_answers_controller.rb

```diff
@@ -66,12 +66,18 @@ def items
     end
   end
 
+  def additional_product_index
+    product_index = nil
+    items.each.with_index { |item, index| product_index = index if item[:field] === t("coronavirus_form.questions.additional_product.title") }
```

Comment:
Not my best piece of work for sure, but don't know how to do it nicer in Ruby.

Reply:
I think you might be able to do something like:

```
product_index = items.index { |item| item[:field] === t("coronavirus_form.questions.additional_product.title") }
```

Reply:
This looks way better! Thank you 🙌